test/Libraries/RevitNodesTests/Elements/Views/AreaPlanViewTests.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Linq;
+using NUnit.Framework;
+using RTF.Framework;
+using Revit.Elements.Views;
+using RevitServices.Persistence;
+using RevitTestServices;
+using Revit.Elements;
+
+namespace RevitNodesTests.Elements.Views
```

Comment:
Can you check if the properties are equal to the input values? Don't just check that they are not null.